pattern-library/sass/patterns/_layouts.scss

```diff
@@ -3,97 +3,29 @@
 
 // About: Contains base settings and options for the preset default Pattern Library UI Layouts.
 
-// #SETTINGS
-// #GLOBAL
-// #UTILITY
 // #CANNED - IN VIEW
 // #CANNED - VIEW
 
-
-// ----------------------------
-// #SETTINGS
-// ----------------------------
 @import 'grid';
 
-$susy-flush:            layout($susy-default inside); // set susy to make columns flush
-$susy-reversed:         layout($susy-default $grid-direction-reversed); // set susy to make columns display in reverse order
-$susy-view:             layout($susy-default auto); // set susy to make columns view display in reverse order
-$susy-view-flush:       layout($susy-default auto inside); // set susy to make view columns display in reverse order
-$susy-view-reversed:    layout($susy-default auto $grid-direction-reversed); // set susy to make view columns display in reverse order
-
-
-// ----------------------------
-// #GLOBAL
-// ----------------------------
-
-
-// ----------------------------
-// #UTILITY
-// ----------------------------
-// CASE: Layout with gutter-separated columns
-%layout-default {
-    @include container($susy-default);
-}
-
-// CASE: Layout with flush columns
-%layout-flush {
-    @include container($susy-flush);
-}
-
-// CASE: Layout with reversed columns
-%layout-reversed {
-    @include container($susy-reversed);
-}
-
-// CASE: View-based layout
-%layout-view {
-    @include container($susy-view);
-}
-
-// CASE: View-based layout with flush columns
-%layout-view-flush {
-    @include container($susy-view-flush);
-}
-
-// CASE: View-based layout with reversed columns
-%layout-view-reversed {
-    @include container($susy-view-reversed);
-}
-
 // basic layout column
-%layout-column {
-    @include span(full); // plan for mobile first
+@mixin layout-column {
+    @include span(12); // plan for mobile first
```

Comment:
I didn't realize that there are layouts that are only provided as demos. It would be good to move demos out into pldoc, and only leave documented patterns here.